pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[tool.poetry]
+name = "peregrine"
+version = "3.0.1"
+description = "Query interface to get insights into data in Gen3 Commons"
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/uc-cdis/peregrine"
+include = [
+  "NOTICE",
+]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+authutils = "3.1.1"
+datamodelutils = "1.0.0"
+defusedxml = "0.5.0"
+dictionaryutils = "3.0.0"
+gen3dictionary = "2.0.1"
+SurvivalPy = "1.0.2"
+simplejson = "3.8.1"
+stopit = "1.1.1"
+python-Levenshtein = "0.12.0"
+python-keystoneclient = "~=1.8"
+fuzzywuzzy = "0.6.1"
+addict = "~=2.1"
+Flask = "1.1.1"
+Flask-Cors = "1.9.0"
+gen3authz = "0.2.3"
+Jinja2 = "2.10.1"
+lxml = "4.4.1"
+MarkupSafe = "0.23"
+PyYAML = "5.1"
+Werkzeug = "0.16.0"
+boto = "2.36.0"
+elasticsearch = "1.2.0"
+itsdangerous = "0.24"
```

Comment:
in other main services repos, when we switched to using poetry we tried to remove unused dependencies. i think many of these could be removed to clean up this file a bit

Reply:
i understand if you consider this out of scope :-)

Reply:
That's a good idea! I'll take a look at cleaning those up 🧹